src/main/scala/dpla/ingestion3/mappers/providers/CdlExtractor.scala

```diff
@@ -81,4 +83,20 @@ class CdlExtractor extends Extractor with JsonExtractionUtils {
     name = Some("California Digital Library"),
     uri = Some(new URI("http://dp.la/api/contributor/cdl"))
   )
+
+  /**
+    * Generates an MD5 checksum from the given providerId
+    * Implemented per provider
+    *
+    * DigestUtils.md5Hex(id)
+    *
+    * @return MD5 hash value
+    */
+  override def mintItemId(json: Any): String = {
+    json match {
+      case json: JValue => Utils.generateMd5(Option(List("cdl", extractString("id")(json)).mkString("--")))
+      // TODO: case _ => throw Exception(?) or some other error
```

Comment:
I think it shouldn't take Any, that seems like too wide open. Maybe it should be oriented toward something like `providerName, stuffToHash` or something?